Tests/interop/net/field/test_instance_fields.py

```diff
@@ -179,26 +179,27 @@ def f22(): v.InstanceSimpleClassField = SimpleClass(540)
         def f23(): v.InstanceSimpleGenericClassField = SimpleGenericClass[str]("string".upper())
         def f24(): v.InstanceSimpleInterfaceField = ClassImplementSimpleInterface(78)
 
-        funcs = [ eval("f%s" % i) for i in range(1, 25) ]
-        if clr.GetClrType(t).IsValueType:
-            for f in funcs:
-                self.assertWarns(RuntimeWarning, 'may result in updating a copy', f)
-        else: 
-            for f in funcs: f()
-            self._test_verify(o)
-            
-            # set values which need conversion.
-            v.InstanceInt32Field = 100
-            self.assertEqual(o.InstanceInt32Field, 100)
+        for i in range(1, 25):
+            f = eval("f%s" % i)
+            if clr.GetClrType(t).IsValueType:
+                self.assertWarnsRegex(RuntimeWarning, 'may result in updating a copy', f)
+            else:
+                f()
+
+        self._test_verify(o)
 
-            v.InstanceInt32Field = 10.01
-            self.assertEqual(o.InstanceInt32Field, 10)
+        # set values which need conversion.
+        v.InstanceInt32Field = 100
```

Comment:
This should be `v.InstanceInt32Field = long(100)`. Looks like the fact it was a `long` somehow got lost when it got ported from IronPython2.

Reply:
Done